src/main/java/io/cdap/plugin/http/sink/batch/HTTPSinkConfig.java

```diff
@@ -55,17 +67,25 @@ public class HTTPSinkConfig extends BaseHttpConfig {
   public static final String CHARSET = "charset";
   public static final String FOLLOW_REDIRECTS = "followRedirects";
   public static final String DISABLE_SSL_VALIDATION = "disableSSLValidation";
-  public static final String NUM_RETRIES = "numRetries";
+  public static final String PROPERTY_HTTP_ERROR_HANDLING = "httpErrorsHandling";
+  public static final String PROPERTY_ERROR_HANDLING = "errorHandling";
+  public static final String PROPERTY_RETRY_POLICY = "retryPolicy";
+  public static final String PROPERTY_LINEAR_RETRY_INTERVAL = "linearRetryInterval";
+  public static final String PROPERTY_MAX_RETRY_DURATION = "maxRetryDuration";
   public static final String CONNECTION_TIMEOUT = "connectTimeout";
   public static final String READ_TIMEOUT = "readTimeout";
   public static final String FAIL_ON_NON_200_RESPONSE = "failOnNon200Response";
```

Comment:
I think we don't need `failOnNon200Response` anymore. Please remove all references of this variable.

Reply:
Removed. dde1cd273c0c4fee6e7c70505a080df0b47f24f6